recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
     - soname-on-windows.patch
 
 build:
-  skip: true  # [(win and py2k)
-  number: 1
+  skip: true  # [(win and py2k)]
```

Comment:
You can just get rid of this line. We stopped python 2

Reply:
ok, thanks!